orders/models.py

```diff
@@ -0,0 +1,41 @@
+from django.db import models
+
+from products.models import Product
+from users.models import User
+from .status_choices import OrderStatusChoices, SupportTicketStatusChoices
+
+
+class Order(models.Model):
+    user = models.ForeignKey(User, on_delete=models.SET_NULL, null=True)
+    order_date = models.DateTimeField(auto_now_add=True)
+    order_status = models.CharField(max_length=20, choices=OrderStatusChoices.choices)
+    total_price = models.DecimalField(max_digits=10, decimal_places=2, default=0)
```

Comment:
instead of adding a field write a property which return sum of selected items price.